src/js/components/awardv2/visualizations/amounts/AggregatedAwardAmounts.jsx

```diff
@@ -23,28 +24,134 @@ const propTypes = {
     jumpToSection: PropTypes.func
 };
 
+const tooltipStateBySpendingCategory = {
+    obligated: "showObligatedTooltip",
+    current: "showCurrentTooltip",
+    potential: "showPotentialTooltip",
+    exceedsCurrent: "showExceedsCurrentTooltip",
+    exceedsPotential: "showExceedsPotentialTooltip"
+};
+
+const createShowAndCloseTooltipMethod = (ctx, category) => {
```

Comment:
Recommend renaming `ctx` to `context` fo readability.

Reply:
@ebdabbs ,

For some reason I get an `eslint` error `no-param-reassign` when I re-name the parameter.... very weird!!

Reply:
Weird ok I'm good to leave it as-is for now